### DISCUSSION TITLE
默认的三栏的argon右侧样式不是太美观

### DISCUSSION THREAD

**Discussion by JIeJaitt** [solstice23/argon-theme#591] [Q&A] [answered]
默认的三栏的argon右侧样式不太好看，如下图：
<img width="457" alt="image" src="https://user-images.githubusercontent.com/77219045/218022699-6b48f5dc-b503-4d01-a0be-77fabc88dad3.png">
我看到用户墙内不少大佬都进行了魔改优化，希望能有一个官方版本的优化出来

**Comment by alongw** [comment]
有没有可能 这就是原版的

**Comment by JIeJaitt** [comment, marked as answer]
同样的原版差距这么大吗？怎么把那个搜索两个字移动到下面呢？

使用经典小工具插件

**Comment by alongw** [comment]
有没有可能 我用的是主题的 你用的是WordPress的

**Reply by JIeJaitt** [reply]
我也是在后台设置的三栏，默认就长这样，找了半天也不知道在哪里调整
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/77219045/218037399-3d8f6609-dc8e-4b18-b4fd-d4872b8dd75c.png">

**Reply by alongw** [reply]
那是你的问题呀，你自己不知道在哪调整 你就来说人家的主题难看

**Reply by ghost** [reply]
哈哈哈哈哈
